src/testdrive/src/action/consistency.rs

```diff
@@ -131,17 +131,13 @@ async fn check_coordinator(state: &State) -> Result<(), anyhow::Error> {
 
 /// Checks that the in-memory catalog matches what we have persisted on disk.
 async fn check_catalog_state(state: &State) -> Result<(), anyhow::Error> {
-    let maybe_disk_catalog = state
-        .with_catalog_copy(|catalog| catalog.state().clone())
-        .await
-        .map_err(|e| anyhow!("failed to read on-disk catalog state: {e}"))?
-        .map(|catalog| catalog.dump().expect("state must be dumpable"));
-    let Some(disk_catalog) = maybe_disk_catalog else {
-        // TODO(parkmycar): Ideally this could be an error, but a lot of test suites fail.
-        tracing::warn!("No Catalog state on disk, skipping consistency check");
-        return Ok(());
-    };
+    #[derive(Debug, Deserialize)]
+    struct CatalogDump {
+        system_parameter_defaults: Option<BTreeMap<String, String>>,
+    }
```

Comment:
Small bike-shed, I think it's bit confusing to call this `CatalogDump` but it only contains a small subset of the catalog dump contents.

Reply:
Yeah, fair. I was envisioning it might one day grow the other catalog fields. But I probably should have called it something more like `RelevantPartsOfCatalogDump`. I'm not going to go back through CI for this if that's okay, since it's green and I'm strapped foer time.